packages/flutter/test/widgets/media_query_test.dart

```diff
@@ -564,4 +564,146 @@ void main() {
     expect(outsideBoldTextOverride, false);
     expect(insideBoldTextOverride, true);
   });
+
+  group('MediaQueryData parameters cannot be null', () {
```

Comment:
From my experience, using groups can get really messy since the code has to be indented the correct number of times and once you get to 2x and 3x number of nested groups, the code becomes very difficult to read.

This is your call since it's just a single level, but I tend to avoid them if I can!

Reply:
If the repo is following the standards, I won't break them. I'll remove the group.